music21/style.py

```diff
@@ -498,7 +498,7 @@ class StyleMixin(common.SlottedObjectMixin):
     '''
     _styleClass = Style
 
-    __slots__ = ('_style', '_editorial')
+    __slots__: Tuple[str, ...] = ('_style', '_editorial')
```

Comment:
does this need to be here, since it derives from a typed class?

Reply:
Wow, good catch! I probably typed them out of order (this one first)

Reply:
Oh, it was to prevent an error in `class Comment`. I'll just add a disable over there. Some multiple inheritance problem I guess.